docs/tasks/access-application-cluster/create-external-load-balancer.md

```diff
@@ -97,20 +99,34 @@ which should produce output like this:
 
 The IP address is listed next to `LoadBalancer Ingress`.
 
-## Annotation to modify the LoadBalancer behavior for preservation of Source IP
-In 1.5, a Beta feature has been added that changes the behavior of the external LoadBalancer feature.
+## Modify the LoadBalancer behavior for preservation of Source IP
+
+Due to the implementation of this feature, the source IP for sessions as seen in the target
+container will *not be the original source IP* of the client. This is the default behavior as
+of Kubernetes v1.7. However, starting in v1.5, an optional beta feature has been added that
+will preserve the client Source IP for GCE/GKE environments. This feature has been promoted
+to GA in v1.7, with below APIs defined:
 
-This feature can be activated by adding the beta annotation below to the metadata section of the Service Configuration file.
+* `Service.Spec.ExternalTrafficPolicy` - It denotes if this Service desires to route
+external traffic to node-local or cluster-wide endpoints. "Local" preserves the client
+source IP and avoids a second hop for LoadBalancer and NodePort type services, but risks
+potentially imbalanced traffic spreading. "Cluster" obscures the client source IP and
+may cause a second hop to another node, but should have good overall load-spreading.
+* `Service.Spec.HealthCheckNodePort` - It specifies the healthcheck nodePort for the
+service. If not specified, HealthCheckNodePort is created by the service api backend
+with the allocated nodePort. Will use user-specified nodePort value if specified by the
+client. Only effects when Type is set to LoadBalancer and ExternalTrafficPolicy is set
```

Comment:
I'm not sure what the meaning of this sentence is. I think you mean:

It is only has an effect when Type is set to "LoadBalancer" and ExternalTrafficPolicy is set to "Local".